core/controllers/ingest.py

```diff
@@ -90,8 +105,12 @@ def populate_db(workbook: dict[str, pd.DataFrame], mappings: tuple[DataMapping])
                      the workbook to the database.
     :return: None
     """
+    reporting_round = workbook["Submission_Ref"].iloc[0]["Reporting Round"]
```

Comment:
By co-incidence I was reading about these lookups earlier, apparently this is slightly faster if you select by column first, as the slice doesn't mix datatypes. Not that it really mattes here for a 1 line df  :D but thought you might like to know. So something like:
`workbook["Submission_Ref"]["Reporting Round"].iloc[0]`

Reply:
ah good to know! looks slightly more readable too